doc/rst/technotes/operatorMethods.rst

```diff
@@ -68,6 +68,27 @@ does not match the type for any other argument.
 Operator methods can be defined on concrete types, generic types, or particular
 instantiations of generic types.
 
+Operator Methods and Classes
+----------------------------
+
+When an operator method is defined on a class, there are some known issues that
+impact its use in an inheritance situation.  For this reason it is recommended
```

Comment:
Just for less passive voice, maybe

Operators that are defined on classes that are inherited from can run into known issues. To avoid these issues, define the class arguments to the operator methods with ``borrowed``, e.g.: